client/src/main/scripts/slipstream.bootstrap.py

```diff
@@ -30,12 +30,20 @@
 import commands
 import traceback
 import getpass
+import time
+import platform
+import re
 
-SLIPSTREAM_CLIENT_HOME = os.path.join(os.sep, 'opt', 'slipstream', 'client')
+SLIPSTREAM_HOME = os.path.join(os.sep, 'opt', 'slipstream')
+SLIPSTREAM_CLIENT_HOME = os.path.join(SLIPSTREAM_HOME, 'client')
+SLIPSTREAM_CLIENT_SETUP_DONE_LOCK = os.path.join(SLIPSTREAM_CLIENT_HOME, 'setup.done.lock')
+
+MACHINE_EXECUTOR_NAMES = ['node', 'orchestrator']
 
 INSTALL_CMD = None
 DISTRO = None
 PIP_INSTALLED = False
+INITD_BASED_DISTROS = ['CentOS', 'CentOS Linux', 'RedHat', 'Ubuntu']
```

Comment:
Agreed.  At the moment, this is very opportunistic behaviour, and assumes that even `upstart` based ones still support init.d.  Lets try to merge this as is, and let me work on this during this sprint.  https://github.com/slipstream/SlipStreamClient/issues/151